data/contrib_data.py

```diff
@@ -2,13 +2,15 @@
 
 import requests
 
-from community.git import get_org_name
-from data.models import Contributor
+# from community.git import get_org_name
+from data.models import Contributor, Team
 
 
 def get_contrib_data():
     logger = logging.getLogger(__name__)
-    IMPORT_URL = 'https://webservices.' + get_org_name() + '.io/contrib/'
```

Comment:
Maybe keep this commented? Would be a newcomer issue then, to just uncomment this afterward. :smile: